plugins/ntuplizer.cc

```diff
@@ -215,6 +234,8 @@ void ntuplizer::beginJob() {
    tree_out->Branch("lumiBlock", &lumiBlock, "lumiBlock/I");
    tree_out->Branch("run", &run, "run/I");
 
+   //tree_out->Branch("primary_Vertex", &PVs.x)
```

Comment:
Can we maybe add the PV in a similar form as is it put here? In case we want to take it as a reference afterwards